### PR TITLE
add onboarding screen

### PR DESCRIPTION
**Pull request by jigneshsolanki**

### CODE REVIEW

**Comment by tiero** [comment]
- Make the frame bit smaller otherwise it's being cut
- Here instead WE NEED the back button (its a pop on the navigator)

image

**Comment by tiero** [comment]
Fix conflicts @jigneshsolanki

File: src/App.tsx [hunk 1]
```diff
@@ -1,80 +1,18 @@
...
-  );
 
-  const closeWelcomeScreen = () => {
-    setDisplayWelcomeScreen(false);
-    sessionStorage.setItem(DISPLAY_WELCOME_SCREEN_KEY, "false");
-  };
-
-  if (!isBrowser && displayWelcomeScreen) {
-    return <WelcomeScreen close={closeWelcomeScreen} />;
+  if (!isBrowser) {
```
Comment (tiero) [inline src/App.tsx:11]: why only in desktop mode?

Comment (jigneshsolanki) [reply src/App.tsx:11]: onboarding is available in desktop mode, please check figma

**Pull request merged**